events/code_review_events/bugbug_utils.py

```diff
@@ -63,23 +70,34 @@ def __init__(self):
         self.hooks_service = taskcluster_config.get_service("hooks")
         self.queue_service = taskcluster_config.get_service("queue")
 
+    async def setup(self):
+        self.task_group_to_build = await EphemeralStorage.create(
+            "bugbug:task_group_to_build", EPHEMERAL_STORAGE_EXPIRATION
+        )
+        self.diff_to_push = await EphemeralStorage.create(
+            "bugbug:diff_to_push", EPHEMERAL_STORAGE_EXPIRATION
+        )
+
     def register(self, bus):
         self.bus = bus
         self.bus.add_queue(QUEUE_BUGBUG)
         self.bus.add_queue(QUEUE_BUGBUG_TRY_PUSH)
 
-    def process_push(self, payload):
+    async def process_push(self, payload):
         mode, build, extras = payload
         if mode != "success":
             return
 
         # Store the push revision and build, so we can use it after bugbug
         # selects tests to add.
-        self.diff_to_push[int(build.diff_id)] = {
-            "revision": extras["revision"],
-            "treeherder_url": extras["treeherder_url"],
-            "build": build,
-        }
+        await self.diff_to_push.set(
+            int(build.diff_id),
```

Comment:
I've just realized I might need to use strings as keys everywhere, let me test...

Reply:
Yes, I needed to do that, and I've done it now.